src/server/editorServices.ts

```diff
@@ -1180,7 +1186,7 @@ namespace ts.server {
                 return { succeeded: false, error: rawConfig.error };
             }
             else {
-                const parsedCommandLine = ts.parseJsonConfigFileContent(rawConfig.config, this.host, dirPath);
+                const parsedCommandLine = ts.parseJsonConfigFileContent(rawConfig.config, this.host, dirPath, /*existingOptions*/ {}, configFilename);

```

Comment:
Do you need to make a similar change at https://github.com/Microsoft/TypeScript/blob/master/src/services/shims.ts#L949 ?